### ISSUE TITLE
awx_task container throws ValueError in awx-manage, invalid port literal

### ISSUE THREAD

**Issue by d-cui** [ansible/awx#5213]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY

I've compiled the AWX application using the installer. When I run `docker logs -f awx_task`, I perpetually get the following error

```
2019-11-01 23:35:18,178 INFO spawned: 'channels-worker' with pid 3598
2019-11-01 23:35:19,180 INFO success: channels-worker entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2019-11-01 23:35:20,443 - INFO - runworker - Using single-threaded worker.
2019-11-01 23:35:20,443 - INFO - runworker - Running worker against channel layer default (asgi_amqp.core.AMQPChannelLayer)
2019-11-01 23:35:20,444 - INFO - worker - Listening on channels websocket.connect, websocket.disconnect, websocket.receive
Traceback (most recent call last):
  File "/usr/bin/awx-manage", line 11, in <module>
    load_entry_point('awx==6.1.0', 'console_scripts', 'awx-manage')()
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/awx/__init__.py", line 142, in manage
    execute_from_command_line(sys.argv)
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/channels/management/commands/runworker.py", line 83, in handle
    worker.run()
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/channels/worker.py", line 87, in run
    channel, content = self.channel_layer.receive_many(channels, block=True)
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/asgiref/base_layer.py", line 43, in receive_many
    return self.receive(channels, block)
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/asgi_amqp/core.py", line 102, in receive
    self._init_thread()
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/asgi_amqp/core.py", line 62, in _init_thread
    self.tdata.connection = kombu.Connection(self.url)
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/kombu/connection.py", line 189, in __init__
    url_params = parse_url(hostname)
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/kombu/utils/url.py", line 47, in parse_url
    scheme, host, port, user, password, path, query = _parse_url(url)
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/kombu/utils/url.py", line 81, in url_to_parts
    parts.port,
  File "/usr/lib64/python3.6/urllib/parse.py", line 169, in port
    port = int(port, 10)
ValueError: invalid literal for int() with base 10: 'ds'
2019-11-01 23:35:20,720 INFO exited: channels-worker (exit status 1; not expected)
```

Any idea what's going on? Am I missing an environment variable?

##### ENVIRONMENT
* AWX version: 6.1.0
* AWX install method: docker on linux
* Ansible version:  2.9.0
* Operating System: Ubuntu 18.04
* Web Browser: N/A

##### STEPS TO REPRODUCE

1. I updated the inventory file, changing the following parameters:
  - Commented out `dockerhub_base=ansible`
  - Changed `postgres_data_dir` to `/opt/awx-psql-data`
  - Changed `ssl_certificate` to `/etc/awx-ssl/awx-bundled-key.crt`
  - Changed docker registry information
  - Changed all passwords
  - Changed `awx_official` to `true`

##### EXPECTED RESULTS

I expected the `awx_task` and `awx_web` containers to start running.

##### ACTUAL RESULTS

The `awx_task` container displays the error above continuously.

##### ADDITIONAL INFORMATION

The same issue happens on AWX 7.0.0.

**Comment by ryanpetrello** [comment]
@d-cui are you using a RabbitMQ password that has an unescaped colon in it (that's my guess, based on the error).

**Comment by wenottingham** [comment]
This should be fixed by https://github.com/ansible/awx/pull/5409, we should have a new release with that out in the near future.

**Comment by ryanpetrello** [comment]
Specifically, this should be resolved in AWX 9.1.0:  

https://github.com/ansible/awx/releases/tag/9.1.0
